compiler/rustc_session/src/config.rs

```diff
@@ -1398,9 +1398,16 @@ pub enum OptionKind {
 }
 
 pub struct RustcOptGroup {
-    apply: Box<dyn Fn(&mut getopts::Options) -> &mut getopts::Options>,
+    /// Display name for this option. Normally equal to `long_name`, except for
+    /// options that don't have a long name.
     pub name: &'static str,
```

Comment:
Suggestion: `display_name`? We have `short_name` and `long_name` below, so having just `name` feels a bit confusing.

Reply:
So, in the process of looking into this, I remembered that things are actually more complicated.

We do use this as a display name, but it also gets used for other things, such as matching against the option name reported by some `getopts` errors.

So I'm going to push a better comment, but now I don't like the idea of trying to rename `name` in this PR.

Reply:
Yes, that's fine with me.